### DISCUSSION TITLE
I am having trouble setting up the app for the clicker app

### DISCUSSION THREAD

**Discussion by Colin-Lee-sst** [sst-inc/help#21] [Q&A - React] [answered]
I don't understand why is there this error when I try setting up my app and what do we submit in the google classroom assignment
<img width="1440" alt="Screenshot 2021-08-04 at 3 34 30 PM" src="https://user-images.githubusercontent.com/88426084/128143230-37becf82-216a-421c-9455-84308190b1a2.png">
![Uploading Screenshot 2021-08-04 at 3.34.30 PM.png…]()

**Comment by ghost** [comment]
Hi Colin, did you open your terminal in the clicker_app folder? It seems that your terminal's working directory is your home folder. Please try using cd to navigate to your clicker_app folder in the terminal and then run expo start again.

You should submit a zip file of your clicker_app folder in the Google Classroom assignment.

**Comment by Colin-Lee-sst** [comment]
Hi Xuan Han,
Sorry for the late reply because I am using my alternate email for github
but could you do a demo and show me how it is done

**Comment by Colin-Lee-sst** [comment]
Hi Xuan Han,
Please ignore the email at the top as I have resolved it but when I opened
it in my web browser, it does not show the app. Is there any way of
resolving this issue[image: Screenshot 2021-08-10 at 11.16.11 AM.png]

**Reply by ghost** [reply]
Hi, I can't seem to be able to view your latest screenshot. Could you send it again?

**Comment by Colin-Lee-sst** [comment]
Ok

**Comment by Colin-Lee-sst** [comment]
[image: Screenshot 2021-08-10 at 11.16.11 AM.png]

**Comment by Ethan-Chew** [comment, marked as answer]
Has this been resolved?

**Comment by Colin-Lee-sst** [comment]
Yes
